azure-pipelines.yml

```diff
@@ -16,11 +16,9 @@ parameters:
       - name: SmokeTest
         command: smoke
         serenity: false
-        dependsOn: UnitTest
```

Comment:
After consulting with Pettedsen, I removed dependencies for functional test, which will now run in parallel. This makes pipeline execution much faster. Consecutive test (security) will still depend on successful execution of all func tests

Reply:
Apologies....slight misunderstanding Kara... as per my email yesterday all the functional dependancies should be run in sequence.

so Dev, Followed by Smoke followed by Acceptance followed by Contract followed by Functional.

You could have dependancy on functional going downwards to Unit... or call them independantly in a sequence...